python/ray/data/datasource/avro_datasource.py

```diff
@@ -5,13 +5,13 @@
 from ray.data.block import Block
 from ray.data.context import DataContext
 from ray.data.datasource.file_based_datasource import FileBasedDatasource
-from ray.util.annotations import PublicAPI
+from ray.util.annotations import DeveloperAPI
 
 if TYPE_CHECKING:
     import pyarrow
 
 
-@PublicAPI(stability="alpha")
+@DeveloperAPI
```

Comment:
To clarify, could we make these internal in the sense of moving them to `ray.data._internal` and removing the decorator altogether?

Reply:
ah i mis-understood that, let me do that instead ;)